Streams/MemoryStreamWriter.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include "StreamWriter.h"
```

Comment:
This should import SeekableStreamWriter, the direct parent. The SeekableStreamWriter class will need to import StreamWriter, so we get that header indirectly.